src/pretalx/static/orga/js/main.js

```diff
@@ -136,9 +136,9 @@ function question_page_toggle_view() {
 }
 
 function question_page_toggle_target_view() {
-  if ($("#limit-submission").length) {
+  if ($(".limit-submission").length) {
     setVisibility(
-      "#limit-submission",
+      ".limit-submission",
```

Comment:
This won't work, I think, unless you modify the ``setVisibility`` function to iterate over the results of ``document.querySelectorAll``. ``document.querySelector`` only returns the first match.

Reply:
Modified the `setVisibility`. I hope we are not constrained to a vanilla JavaScript and `$` is an option. Let me know what you think.